apps/meeting/src/app.tsx

```diff
@@ -57,6 +42,44 @@ const App: FC = () => (
   </Router>
 );
 
+const MeetingView: React.FC = () => {
+  const { addVoiceFocus } = useVoiceFocus();
+  const { meetingMode } = useAppState();
+  const [isVoiceFocusChecked, setIsVoiceFocusChecked] = useState(false);
+
+  const reselection = (device: Device): Promise<Device | VoiceFocusTransformDevice> => {
+    if (isVoiceFocusChecked) {
+      return addVoiceFocus(device);
+    }
+    return Promise.resolve(device);
+  };
+
+  const toggleVoiceFocus = () => {
+    setIsVoiceFocusChecked(current => !current);
+  }
```

Comment:
Remember our previous discussions about how this kind of state management isn't accurate, because applying Voice Focus can fail?